src/App.jsx

```diff
@@ -31,6 +31,12 @@ function App() {
     window.location.hostname === "nucleoid.com" ? 1000 - elapsed : 0;
 
   function initSettings() {
+    const isMobile = window.innerWidth <= 768;
```

Comment:
We better use MUI API in this
https://github.com/NucleoidAI/IDE/pull/206/commits/e0aa9c2e00c2c12f13fdb7068095b39fbc2b9b4a

Reply:
We not use it in Chat container